flink-connectors/flink-connector-base/src/main/resources/log4j2.properties

```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 ################################################################################
 
-rootLogger.level = INFO
+rootLogger.level = OFF
```

Comment:
Why did you change this?

Reply:
Fabian had recommended it in #17345. :-)